src/components/client/pages/categories/components/Atoms/recipeAtoms/RecipeImageAtom.module.css

```diff
@@ -0,0 +1,4 @@
+.RecipeImageAtom {
+  border-radius: 0.5rem;
```

Comment:
@JakubLawnicki for border-radius it is not recommended to convert the px value to rem. 💡